pephub/routers/api/v1/project.py

```diff
@@ -1138,3 +1143,61 @@ def delete_full_history(
             status_code=400,
             detail="Could not delete history. Server error.",
         )
+
+
+@project.post(
+    "/standardize",
+    summary="Standardize PEP metadata column headers",
+    response_model=StandardizerResponse,
+)
+async def get_standardized_cols(
+    pep: peppy.Project = Depends(get_project),
+    schema: str = "",
+):
+    """
+    Standardize PEP metadata column headers using BEDmess.
+
+    :param namespace: pep: PEP string to be standardized
+    :param schema: Schema for AttrStandardizer
+
+    :return dict: Standardized results
+    """
+
+    if schema == "":
+        raise HTTPException(
+            code=500,
+            detail="Schema is required! Available schemas are ENCODE and Fairtracks",
+        )
+        return {}
+
+    prj = peppy.Project.from_dict(pep)
+    model = AttrStandardizer(schema)
+
+    try:
+        results = model.standardize(pep=prj)
+    except Exception:
+        raise HTTPException(
+            code=400,
+            detail=f"Error standardizing PEP.",
+        )
+
+    return StandardizerResponse(results=results)
+
+
+@project.get(
+    "/history/{history_id}/zip",
+    summary="Zip a project history by id",
+    response_class=FileResponse,
+)
+def get_zip_snapshot(
+    namespace: str,
+    project: str,
+    history_id: int,
+    tag: str = DEFAULT_TAG,
+):
+    """
+    Get a project dict from history by id
+    """
+
+    schemas = model.show_available_schemas()
```

Comment:
This logic doesnt seem to fit what the endpoint actually is trying to do.

Reply:
Seems to me that it should be fetching a ZIP archive of a project given a history id, but it just returns schemas

Reply:
yeah, it seems incorrect

Reply:
Removed, this was a copy paste remnant